docs/src/modules/components/AppLayoutDocsFooter.js

```diff
@@ -308,7 +308,7 @@ export default function AppLayoutDocsFooter() {
             </PaginationDiv>
           </React.Fragment>
         )}
-        <Collapse in={commentOpen} onEntered={handleEntered}>
+        <Collapse in={commentOpen} unmountOnExit onEntered={handleEntered}>
```

Comment:
I would extract this to a separate PR, but probably not worth the effort :D

Reply:
I see what you mean. What's the main value proposition of this PR? To be fair, I think that the WAVE error is wrong and should be ignored, the input is not interactive. The real value is about reducing the upfront work to render the page. 

This win:

<img width="193" alt="Screenshot 2021-12-24 at 22 28 16" src="https://user-images.githubusercontent.com/3165635/147372651-8f2bf5b2-754c-4125-8478-0cb710bbbf6d.png">

is a good side effect. But if you go to https://mui.com/components/textarea-autosize/, you will still see the error.